motionplan/constraint.go

```diff
@@ -230,118 +326,70 @@ func newCollisionConstraint(
 	}
 
 	// create constraint from reference collision graph
-	constraint := func(cInput *ConstraintInput) (bool, float64) {
-		internal, err := cInput.Frame.Geometries(cInput.StartInput)
+	constraint := func(cInput *StateInput) bool {
+		internal, err := cInput.Frame.Geometries(cInput.Configuration)
 		if err != nil && internal == nil {
-			return false, 0
+			return false
 		}
 
 		cg, err := newCollisionGraph(internal.Geometries(), static, zeroCG, reportDistances)
 		if err != nil {
-			return false, 0
+			return false
 		}
 
 		collisions := cg.collisions()
-		if len(collisions) > 0 {
```

Comment:
We need to add this back now right?

Reply:
what is `reportDistances` doing as an argument if we are not doing this anymore?

Reply:
Once the `PenetrationDepthMetric` that I mentioned in my TODO is written, that will use `true`